docs/nf4_science/imaging/01_basics.md

```diff
@@ -0,0 +1,413 @@
+
+There are two key advantages to doing this:
+
+- If you're in the middle of developing a pipeline, you can iterate more rapidly since you only have to run the process(es) you're actively working on in order to test your changes.
+- If you're running a pipeline in production and something goes wrong, in many cases you can fix the issue and relaunch the pipeline, and it will resume running from the point of failure, which can save you a lot of time and compute.
+
+To use it, simply add `-resume` to your command and run it:
+
+```bash
+nextflow run hello-world.nf --greeting 'Hello World!' -resume
+```
+
+The console output should look similar.
+
+<details>
+  <summary>Command output</summary>
+
+```console linenums="1"
+ N E X T F L O W   ~  version 25.04.3
+
+Launching `hello-world.nf` [tiny_noyce] DSL2 - revision: c33d41f479
+
+[a3/7be2fa] process > sayHello [100%] 1 of 1, cached: 1 ✔
+```
+
+</details>
+
+Look for the `cached:` bit that has been added in the process status line (line 5), which means that Nextflow has recognized that it has already done this work and simply reused the result from the previous successful run.
+
+You can also see that the work subdirectory hash is the same as in the previous run.
+Nextflow is literally pointing you to the previous execution and saying "I already did that over there."
+
+!!! Tip
+
+    When your re-run a pipeline with `resume`, Nextflow does not overwrite any files written to a `publishDir` directory by any process call that was previously run successfully.
+
+### 3.2. Inspect the log of past executions
+
+Whenever you launch a nextflow workflow, a line gets written to a log file called `history`, under a hidden directory called `.nextflow` in the current working directory.
+
+A more convenient way to access this information is to use the `nextflow log` command.
+
+```bash
+nextflow log
+```
+
+This will output the contents of the log file to the terminal, showing you the timestamp, run name, status, and full command line for every Nextflow run that has been launched from within the current working directory.
+
+### 3.3. Delete older work directories
+
+During the development process, you'll typically run your draft pipelines a large number of times, which can lead to an accumulation of very many files across many subdirectories.
+Since the subdirectories are named randomly, it is difficult to tell from their names what are older vs. more recent runs.
+
+Nextflow includes a convenient `clean` subcommand that can automatically delete the work subdirectories for past runs that you no longer care about, with several [options](https://www.nextflow.io/docs/latest/reference/cli.html#clean) to control what will be deleted.
+
+You can use the Nextflow log to look up a run based on its timestamp and/or command line, then use `nextflow clean -before <run_name> -f` to delete work directories from earlier runs.
```

Comment:
```suggestion
You can use the Nextflow log to look up a run based on its timestamp and/or command line, then use `nextflow clean -before tiny_noyce -f` (the run name from the previous example) to delete work directories from earlier runs.
```

Reply:
I actually like the generic term better in this case.